app.py

```diff
@@ -1,19 +1,29 @@
 # app.py
 from flask import Flask, request, jsonify
 from validator import validate_login
+from flask_cors import CORS
+from flask.logging import create_logger
+import sqlite3
+import uuid
+#import UserInfo from './userinfo'
 
 app = Flask(__name__)
+create_logger(app)
 
 @app.route('/login/', methods=['POST'])
 def login():
+    if not request.form:
+        app.logger.error(f"could not parse a request: {request.data}")
     username = request.form.get("username", None)
     password = request.form.get("password", None)
+    app.logger.warning(f"received username {username}, password {password}, entirepayload {request.form}")
```

Comment:
Just a reminder, change this to a lower log level later (TODO)
debug - info - warning - error